bin/cheritest/cheritest_fault.c

```diff
@@ -261,4 +261,21 @@ test_fault_read_epcc(const struct cheri_test *ctp __unused)
 
 	CHERI_CAP_PRINT(cheri_getepcc());
 }
+
+void
+test_fault_cloadtags_misaligned(const struct cheri_test *ctp __unused)
+{
+	uint64_t tags = 0xDEADFEED;
+	void * __capability p[20] __attribute__((aligned(512))) = { 0 } ;
+	void * __capability c = &p[1];
+
+	__asm__ ("cloadtags %[tags], %[c]"
+		: [tags]"=r"(tags)
+		: [c]"C"(c)
+		: "memory");
```

Comment:
Hm, should we not be using `__builtin_cheri_cap_load_tags` for all of these rather than adding back inline assembly? We've managed to get rid of basically all inline assembly from cheritest.

Reply:
Oh good call.  These tests predate the builtin, I think.